classes/plugins/ThemePlugin.inc.php

```diff
@@ -33,31 +33,23 @@ function register($category, $path) {
 
 		$request = $this->getRequest();
 		if ($result && $this->getEnabled() && !defined('SESSION_DISABLE_INIT')) {
-			HookRegistry::register('PageHandler::displayCss', array($this, '_displayCssCallback'));
 			$templateManager = TemplateManager::getManager($request);
+			HookRegistry::register('PageHandler::displayCss', array($this, '_displayCssCallback'));
+
+			// Add the stylesheet.
 			$dispatcher = $request->getDispatcher();
 			$templateManager->addStyleSheet($dispatcher->url($request, ROUTE_COMPONENT, null, 'page.PageHandler', 'css', null, array('name' => $this->getName())), STYLE_SEQUENCE_LATE);
+
+			// If this theme uses templates, ensure they're given priority.
+			array_unshift(
+				$templateManager->template_dir,
+				$path = Core::getBaseDir() . DIRECTORY_SEPARATOR . $this->getPluginPath() . '/templates'
```

Comment:
I went on a rampage some years ago replacing "/" with DIRECTORY_SEPARATOR. It's way less readable and as far as I know has never been necessary. We're currently using a mix of the two, and I grabbed this from someplace that got rampaged. Generally speaking, "/" is a lot nicer to read, but mixing hasn't been a problem.